aten/src/ATen/cuda/nvrtc_stub/ATenNVRTC.h

```diff
@@ -84,7 +84,10 @@ namespace at { namespace cuda {
   _(nvrtcCompileProgram)                         \
   _(cuCtxGetCurrent)                             \
   _(cuModuleUnload)                              \
-  _(cuDevicePrimaryCtxGetState)
+  _(cuDevicePrimaryCtxGetState)                  \
```

Comment:
Nope, we don't want this for rocm. It will break rocm build, as their driver API doesn't provide these.

This is something that we fixed in last PR.

Reply:
Removed, thanks for catching that.